### DISCUSSION TITLE
How do I use count with download-open?

### DISCUSSION THREAD

**Discussion by Aster89** [qutebrowser/qutebrowser#6501] [Q&A] [answered]
I haven't ever understood this, and probably not needed it much, but I guess it's time I do.

Hitting the number in normal model as I'd do in Vim and then going for `:download-open`<kbd>Enter</kbd> doesn't seem to work.

**Comment by The-Compiler** [comment, marked as answer]
Using counts like this only works for commands that are bound to a key. If you had `download-open` bound to, say, `,o`, you could then hit `2,o`. If you're using the command line, you'd need to use the `run-with-count` command instead.
